app/views/devise/registrations/new.html.haml

```diff
@@ -53,5 +55,9 @@
           .user-alert.hidden
             = image_tag 'icons/exclamation.png', alt: '!', title: '', class: 'vmid'
             %span.msg
-    %tr
-      %th.form-table-ender{colspan: 2}= submit_tag "Sign Up", class: 'button'
+    %tfoot
+      %tr
+        %th.form-table-ender{colspan: 2}
+          = submit_tag "Sign Up", class: 'button'
+          %br
+          = render "devise/shared/links"
```

Comment:
What's this do?

Reply:
Right now, all it does is look ugly. :D 

I'm still looking into how I best want to format these, but ideally it lets you access the "hey I didn't get an email confirmation" page which will be new as part of Devise, and also lets you switch between the "Sign up" and "Log in" pages (though I'm not sure we need that one: we have links in the header for that).

I think it might be enough to just insert the email confirmation link without the other links here.